types.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+type Status string
+
+const (
+	StatusError Status = "error"
+	StatusWarn  Status = "warn"
+	StatusInfo  Status = "info"
+)
+
+func (s Status) getColorCode() int {
+	switch s {
+	case StatusError:
+		return 0xFF4C4C
+	case StatusWarn:
+		return 0xFFA500
+	case StatusInfo:
+		return 0x4CAF50
+	}
+
+	return -1
+}
+
+func (s Status) validStatus() bool {
```

Comment:
```suggestion
func (s Status) isValidNotificationLevel() bool {
```
이 `is**` prefix 를 붙이는게 사용하는쪽에서 더 직관적일 것 같습니다.

Reply:
네넵, 제안주신 prefix가 더 직관적일 거 같습니다.
`isValidNoticeLevel`로 변경하였습니다. 감사합니다

[644a9b6](https://github.com/themoment-team/hellogsm-notice-server/pull/1/commits/644a9b6144d7d6b58d780c04b6aa3464cd469622)